finch/processes/utils.py

```diff
@@ -62,10 +63,28 @@
     "data_validation",
 ]
 
+
+def get_virtual_modules():
+    """Load virtual modules."""
+    modules = {}
+    if modfiles := get_config_value("finch", "xclim_modules"):
+        modfolder = [
+            f for f in Path(".").rglob("*") if f.is_dir() and f.name == "modules"
+        ][0]
+        for modfile in modfiles.split(";"):
+            mod = build_indicator_module_from_yaml(modfolder.joinpath(modfile))
```

Comment:
indeed. fogot I did this...  was having trouble with relative paths and just wanted to get it working

Reply:
Is this closer to what you would have in mind?
````
    modules = {}
    if modfiles := get_config_value("finch", "xclim_modules"):
        for modfile in modfiles.split(","):
            if os.path.isabs(modfile):
                mod = build_indicator_module_from_yaml(Path(modfile))
            else:
                mod = build_indicator_module_from_yaml(Path("finch").joinpath(modfile))
            indicators = []
            for indname, ind in mod.iter_indicators():
                indicators.append(ind.get_instance())
            modules[Path(modfile).name] = dict(indicators=indicators)
    return modules
````

Otherwise the config could map `finch/processes/modules/humidex;finch/processes/modules/streamflow`  with no need for an `if` statement

Reply:
have implemented as above for now

Reply:
Yes, that was what I had in mind!

However, instead of `Path("finch")`, maybe: `Path(__file__).parent.parent` ? The former depends on the current working directory.

Reply:
done